ui/src/lib/components/Form.svelte

```diff
@@ -51,41 +56,43 @@
             <input
                 type="text"
                 on:input={handlePathInput}
-                bind:value={formData.pathText}
+                bind:value={formData.path}
                 placeholder="C:/images"
+                autocomplete="on"
             />
         </label>
-        {#if wrongPath && formData.pathText !== ''}
+        {#if wrongPath && formData.path !== ''}
             <p class="errorMessage">This is not a path</p>
         {/if}
     </div>
     <div class="inputContainer">
         <p class="inputContainerTitle" on:click={handleMetadataClick}>
+            <input type="checkbox" bind:checked={formData.metadata.active} on:click|stopPropagation/>
             Metadata <i class="arrow {arrowDirection}" />
         </p>
         {#if metadataVisible}
             <div class="moduleForm" transition:slide>
-                <label
+                <!-- <label
                     >Author
-                    <input type="text" bind:value={formData.author} placeholder="John Smith" />
+                    <input type="text" bind:value={formData.metadata.author} placeholder="John Smith" />
                 </label>
                 <label
                     >Extension
-                    <input type="text" bind:value={formData.extension} placeholder="png" />
-                </label>
+                    <input type="text" bind:value={formData.metadata.extension} placeholder="png" />
+                </label> -->
```

Comment:
Will this commented code be important later or is it just a trash?
```suggestion
```

Reply:
Now they aren't necessary, but will be for the person who will be adding text inputs to have example of them.